lib/breakers/statsd_plugin.rb

```diff
@@ -11,7 +11,14 @@ def get_tags(request)
           # * (possibly negative) digit identifiers
           # * uuid's with or without dashes
           # * institution id's of form 111A2222 or 11A22222
-          r = %r{(\/)(\-?\d+|[a-fA-F0-9]{8}(\-?[a-fA-F0-9]{4}){3}\-?[a-fA-F0-9]{12}|[\dA-Z]{8})(\/|$)}
+          r = %r{
```

Comment:
What about breaking the regex down into parts and interpolating?  This regex is a beast. 
````
 useful_descriptor = %r{[a-fA-F0-9]{8}(\-?[a-fA-F0-9]{4}){3}\-?[a-fA-F0-9]{12}}
          r = %r{
            (\/)
            (\-?\d+|
              \d{10}V\d{6}%5ENI%5E200M%5EUSVHA|
              #{useful_descriptor}|
              [\dA-Z]{8})
            (\/|$)
          }x
````